plugins/vfswad/vfs.cpp

```diff
@@ -119,9 +119,7 @@ int vfsBuildShortPathName( const char* pPath, char* pBuffer, int nBufferLen ){
 		strcpy( pBuffer, pPath );               // Use long filename
 	}
 	return nResult;
-#endif
-
-#if defined ( __linux__ ) || defined ( __APPLE__ )
+#elif defined ( __linux__ ) || defined ( __APPLE__ )
```

Comment:
There is no reason for this code to be pinned to only linux and macos. An `#else` would be more appropriate.